src/Settings/Settings.ts

```diff
@@ -76,6 +76,8 @@ export const Settings = {
   TailRenderInterval: 1000,
   /** Theme colors. */
   theme: { ...defaultTheme },
+  /** Theme Name (needed for neofetch) */
+  themeName: "Default",
```

Comment:
I don't think this is worth adding just for `neofetch`. Also, since you can have custom themes it's not even guaranteed that your theme will have a name.

Reply:
Good point. I can remove this as it will display some colors from the theme in the future anyway. I have been thinking about submitting a pr for the theme menu allowing you to save themes with names, so if I get to it I can implement this there.